### DISCUSSION TITLE
Veros Beginner's

### DISCUSSION THREAD

**Discussion by claudioiturra** [team-ocean/veros#617] [General]
Hi, I'm totally new to this software and i want to  learn in the right way with the purpose to simulate inertial oscillation in the coastal ocean, like in a very short domain. I will appreciate any advise in how to learn with examples o reading material. Do i need a mesh of my study area, basic wind data, etc? again, thank you so much for your time.

**Comment by dionhaefner** [comment]
We currently don't have a tutorial on setting up a new setup from scratch. Your best bet is to look at our [available examples](https://github.com/team-ocean/veros/tree/main/veros/setups) and modify one to fit your needs.
